### DISCUSSION TITLE
Validator Questions

### DISCUSSION THREAD

**Discussion by CryptoDevving** [0xPolygon/polygon-edge#819] [General]
Am I able to get non-bootnodes to be validators or are bootnodes the only thing that can validate the blockchain? 

Also if I am able to have non-bootnodes to be validators am I able to have validators require 100 coins to be validators?

I am using the latest version
Ubuntu 22

**Comment by laviniat1996** [comment]
Hi @CryptoDevving ,

Not all validators are bootnodes. You add how many bootnodes you want manually when you generate the genesis file (check our [local setup guide](https://wiki.polygon.technology/docs/edge/get-started/set-up-ibft-locally#step-3-generate-the-genesis-file-with-the-4-nodes-as-validators) ).

I'm afraid I don't fully understand the question about 100 coins.

**Reply by CryptoDevving** [reply]
Yes. I already started the chain. Is there anyway to add validators on a public chain without them being bootnodes? 

I want people to stake 100 coins to be a validator. Similarly to how AVAX has people stake 2000 AVAX to be a validator.

**Reply by laviniat1996** [reply]
Bootnodes are added by using the `--bootnode ` genesis flag. Example: -`-bootnode /ip4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW` . When you generate the keys for a validator, it doesn't automatically become a bootnode. 

As you can find in our [docs](https://wiki.polygon.technology/docs/edge/faq/validators#is-the-minimum-staking-amount-configurable-for-pos-edge) also, the minimum staking amount for PoS Edge is `1 unit of native currency`, and it is not configurable.

**Reply by CryptoDevving** [reply]
Ok. I am unable to change the minimum for the staking? I cannot make it 1000 or something?

EDIT: Are people able to stake to become only a validator and not a bootnode?

EDIT 2: How can I access the coins I get as a validator bootnode?

**Reply by CryptoDevving** [reply]
@laviniat1996

**Reply by laviniat1996** [reply]
@CryptoDevving 

1. Yes, you cannot change the minimum amount.
2. Yes, you can stake without becoming a bootnode. Bootnodes are added manually, not every validator is a bootnode.
I'll get back with a reply for the 3rd question tomorrow. Thank you for your patience.

**Reply by CryptoDevving** [reply]
So how can I add a validator without being a bootnode?
@laviniat1996

**Reply by laviniat1996** [reply]
@CryptoDevving You can access the rewards the validator gets via it's private key. Those tokens are stored to the validator's address, and you can access that account via it's private key.

You can find here how to become a validator in PoS: https://wiki.polygon.technology/docs/edge/consensus/pos-concepts#staking

**Reply by CryptoDevving** [reply]
@laviniat1996 are you able to mint new coins from bootnodes? Say use a bootnode to send someone 1000 coins?

**Reply by laviniat1996** [reply]
@CryptoDevving A bootnode is just a normal node that, when you set it, will be contacted to provide information on how to connect with all remaining nodes in the network. Any node can be a bootnode and you choose how many bootnodes you want. You can premine a certain amount of tokens to a node, and then send to whatever address you'd like.

**Reply by CryptoDevving** [reply]
@laviniat1996 somehow when I send coins it is not working. It becomes a stuck transaction. Why is the transaction stuck if it is producing blocks?

**Reply by laviniat1996** [reply]
@CryptoDevving How are you sending the tx? Via Metamask?  Do you get any error when you send it? A screenshot would be useful.

**Reply by CryptoDevving** [reply]
@laviniat1996 I am sending the transactions via meta mask. Does that have something to do with it? They just say pending. I set the gas fee to super low. Like .000000001 and it still sometimes is stuck.

**Reply by laviniat1996** [reply]
@CryptoDevving You've set the gas fee to super low in Metamask? If so, that's probably why they are stuck.

**Reply by CryptoDevving** [reply]
No. I set it low when I set up polygon edge. 

How do I unstuck them if they are stuck?

@laviniat1996
